README.md

```diff
@@ -421,6 +421,7 @@ Your contributions and suggestions are heartily♥ welcome. (✿◕‿◕). Plea
 * [Gophish](https://getgophish.com) - Open-source phishing framework.
 * [phishery](https://github.com/ryhanson/phishery) - TLS/SSL enabled Basic Auth credential harvester.
 * [ReelPhish](https://github.com/fireeye/ReelPhish) - Real-time two-factor phishing tool.
+* [Modlishka](https://github.com/drk1wi/Modlishka) - Flexible and powerful reverse proxy. It allows real-time two-factor authentication.
```

Comment:
Can we make the description one sentence? Maybe `Flexible and powerful reverse proxy with real-time two-factor authentication.`

Reply:
Description changed. Ty